Kconfig

```diff
@@ -200,6 +200,14 @@ config OPTIMIZE_FOR_NONE
 
 endchoice
 
+config OPTIMIZE_FLAGS
+	string
+	default "O2" if OPTIMIZE_FOR_PERFORMANCE
+	default "Os" if OPTIMIZE_FOR_SIZE
+	default "Og" if OPTIMIZE_FOR_DEBUG
+	default "O0" if OPTIMIZE_FOR_NONE
+	default "OX"
+
```

Comment:
These are arch/compiler specific and cannot be in Kconfig

Reply:
It is optimization description passed to cc_version struct and log in host system.
Moreover as long as we use compiler based on GCC such a solution fit their optimization flags values,
but it don't have to. When there will be some compiler with non-standard optimization flags interface
then we can handle it in old way for this arch/compiler without any impact on the rest.